it/mail/chrome/messenger/virtualFolderListDialog.dtd

```diff
@@ -1,4 +1,2 @@
-<!ENTITY virtualFolderListTitle.title        "Selezionare la/e cartella/e">
-<!ENTITY virtualFolderDesc.label       "Selezionare le cartelle in cui cercare:">
-
-
```

Comment:
@albertovlcbo Let's be careful about making changes that are not necessary, as it creates more work for people to review.